velox/common/caching/StringIdMap.cpp

```diff
@@ -60,11 +60,11 @@ uint64_t StringIdMap::makeId(std::string_view string) {
   if (it != stringToId_.end()) {
     auto entry = idToEntry_.find(it->second);
     VELOX_CHECK(entry != idToEntry_.end());
-    if (++entry->second.numInUse == 1) {
-      pinnedSize_ += entry->second.string.size();
```

Comment:
So pinnedSize_ update is redundant?

Reply:
@majetideepak  `++entry->second.numInUse` is on L64 of the new file.